ngx-fudis/projects/ngx-fudis/src/lib/components/alert/alert.mdx

```diff
@@ -9,6 +9,12 @@ import { AlertGroupComponent } from "./alert-group/alert-group.component.ts";
 
 Alert Group Component displays list of toaster-like Alert Components with four variants: `success`, `info`, `warning` and `danger.` These variants are the same ones as with [Notification Component](/docs/components-notification--documentation).
 
+## Usage Guidelines
+
+First, add `<fudis-alert-group />` HTML tag in the application template, preferably somewhere up in the application's DOM, e.g. `app.component.html`.
```

Comment:
For the DOM structure we should probably instruct to put component as the "bottom element" of app.component.html etc. as that will make it more likely, that it will appear as the top element, even top of Navigation. This probably should be fixed in our examples and in our Dialog component as well, that Alert Group is last and not first element in DOM.